tests/test_tiatoolbox.py

```diff
@@ -32,15 +34,15 @@ def _response_ndpi(tmpdir_factory):
     """
     ndpi_file_path = tmpdir_factory.mktemp("data").join("CMU-1.ndpi")
     if not os.path.isfile(ndpi_file_path):
-        print("Downloading NDPI")
+        print("\nDownloading NDPI")
```

Comment:
Curious as to why you are adding an additional line break here (and in other prints), especially as it just used for debugging and won't show up normally.

Reply:
It was looking weired on my screen so I just added it.